### PR TITLE
Kosagi palawan

### PR DESCRIPTION
**Pull request by xobs**
This makes a request for the Kosagi Palawan bit-banged USB stack.

This project will be presented at 33c3 in December.

### CODE REVIEW

**Comment by Arachnid** [comment]
Thanks for your PR! Can you add license files to your repositories, so users know what license they're under?

**Comment by xobs** [comment]
I've added a LICENSE.md file to joyboot.  Looks like I forgot to push the LICENSE.md file to palawan-hw.  Both should have files now.

File: 1209/9317/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: Palawan-Tx
+owner: Kosagi
+license: CC4.0-BY-SA (hardware) / MUT (firmware)
```
Comment (peternewman) [inline 1209/9317/index.md]: Is this a typo then @xobs , should it be MIT not MUT?

Comment (xobs) [inline 1209/9317/index.md]: Er, yes, that's a typo.  Would you fix it, or should I fix it and submit another PR?

Comment (peternewman) [inline 1209/9317/index.md]: I've not got direct write access (although @Arachnid did offer), I'd either do a PR or see if he offers to edit it in-place.
